resource-watch/resource-watch
# Explore – My Data

## Overview
![image](https://user-images.githubusercontent.com/999124/114676157-4c948380-9d09-11eb-91f5-872c59cdfaee.png)

Adds a new _My Data_ tab in Explore. This tab will allow the user to see its datasets under some conditions:
- only GEE/CartoDB providers are valid.
- the dataset must contain, at least, one layer.

## Testing instructions
Go to `/data/explore`. In the sidebar, you should see a _My Data_ tab.
- if you're not logged, you will see a coming soon view.
- if you're logged, but don't have valid datasets, you will see a coming soon view.
- if you're logged and have valid datasets, you will see those valid datasets.

## Jira task
https://vizzuality.atlassian.net/browse/RW-7

---

## Checklist before submitting
